src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,14 @@
+$day_width: 100px;
+$gap: 1px;
+$calendar_width: 7 * $day_width + 6 * $gap;
+
+$margin-left: $day_width + $gap;
+
+$week:
+  "monday" 0,
+  "tuesday" $margin-left,
+  "wednesday" $margin-left * 2,
+  "thursday" $margin-left * 3,
+  "friday" $margin-left * 4,
+  "saturday" $margin-left * 5,
+  "sunday" $margin-left * 6;

```

Comment:
Be more attentive to details and fix it
![image](https://user-images.githubusercontent.com/57747481/173769948-09ba3b58-4c82-4e51-a04d-9879977051f3.png)